Adafruit_ILI9341.cpp

```diff
@@ -601,3 +601,25 @@ void Adafruit_ILI9341::fillRect(int16_t x, int16_t y, int16_t w, int16_t h,
     writeFillRect(x,y,w,h,color);
     endWrite();
 }
+
+// This code was ported/adapted from https://github.com/PaulStoffregen/ILI9341_t3
+// by Marc MERLIN. See examples/pictureEmbed to use this.
+void Adafruit_ILI9341::drawBitmap(int16_t x, int16_t y, int16_t w, int16_t h, const uint16_t *pcolors) {
+        startWrite();
+	setAddrWindow(x, y, w, h);
+	for(y=h; y>0; y--) {
```

Comment:
My apologies, I missed that bit in your comment. Fixed, tested, and it works fine with writePixels